habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -134,6 +134,7 @@ class ArmRelPosAction(RobotAction):
     def __init__(self, *args, config, sim: RearrangeSim, **kwargs):
         super().__init__(*args, config=config, sim=sim, **kwargs)
         self._delta_pos_limit = self._config.delta_pos_limit
+        self._arm_joint_mask = self._config.arm_joint_mask
```

Comment:
I am concerned that you are adjusting and adding complexity to `ArmRelPosAction` which is used in _all_ rearrangement tasks. Please either separate this into another Action or run training for fully train a pick policy for Fetch in ReplicaCAD (see `rl_skill.yaml` config) to confirm that (1) training throughput is not affected and (2) performance is not impacted.

Reply:
Thank you, @ASzot I will add another action space to prevent complexity.

Reply:
done!